server/go.mod

```diff
@@ -158,6 +158,7 @@ require (
 	github.com/levigross/exp-html v0.0.0-20120902181939-8df60c69a8f5 // indirect
 	github.com/magiconair/properties v1.8.7 // indirect
 	github.com/mailru/easyjson v0.7.7 // indirect
+	github.com/mattermost/go-i18n v1.11.1-0.20211013152124-5c415071e404 // indirect
```

Comment:
@isacikgoz this should no happen after my PR or I did miss something / things have evolved since my october PR. I'll review dependencies and try to remove new go-i18n v1 clients